pandas/core/arrays/string_.py

```diff
@@ -282,8 +284,25 @@ def astype(self, dtype, copy=True):
         return super().astype(dtype, copy)
 
     def _reduce(self, name, skipna=True, **kwargs):
+        if name in ["min", "max"]:
+            return getattr(self, name)(skipna=skipna)
+
         raise TypeError(f"Cannot perform reduction '{name}' with string dtype")
 
+    def min(self, skipna=True, **kwargs):
+        nv.validate_min((), kwargs)
+        result = masked_reductions.min(
+            values=self.to_numpy(), mask=self.isna(), skipna=skipna
```

Comment:
isnt the point of masked_reductions that we _dont_ have to do to_numpy?

Reply:
I was following the annotations there, but taking another look I don't see why values has to be an ndarray. Maybe those types could be updated?

Reply:
`masked_reductions.py` is purely numpy. You pass it the numpy array parts of the EA. 
This could also be `self._ndarray` instead of `self.to_numpy()`, but both are exactly the same